tests/unit/model/message_test.py

```diff
@@ -4287,3 +4304,19 @@ def test_refinement_invalid_condition_unqualified_literal() -> None:
         r' of "P::M"'
         r"$",
     )
+
+
+def test_boolean_value_as_condition() -> None:
```

Comment:
Nit: This is actually testing a boolean variable and not a boolean value (`True`, `False`).

Reply:
Fixed.